junit-jupiter-api/src/main/java/org/junit/jupiter/api/Assertions.java

```diff
@@ -1169,6 +1169,23 @@ public static void assertAll(String heading, Stream<Executable> executables) thr
 		return AssertThrows.assertThrows(expectedType, executable);
 	}
 
+	/**
+	 * <em>Asserts</em> that execution of the supplied {@code executable} throws
+	 * an exception of the {@code expectedType} and returns the exception.
+	 *
+	 * <p>If no exception is thrown, or if an exception of a different type is
+	 * thrown, this method will fail.
+	 *
+	 * <p>If you do not want to perform additional checks on the exception instance,
+	 * simply ignore the return value.
+	 *
+	 * <p>If the given {@link ThrowingSupplier<?>} returns a result instead of throwing the expected exception,
```

Comment:
Checkstyle choked on the `<` character. If I'm not mistaken you may omit the `<?>`.